src/main/webapp/js/components/csv/csv-config-create.jsx

```diff
@@ -30,11 +31,30 @@ class CsvConfigCreate extends Component {
 
     this.state = {
       entity: '',
+      initialValues: {},
```

Comment:
I think you can rename, the initialValues to initialFields, for the name to be more descriptive